_posts/2019-08-06-pre-post-events-sustainable-trains.md

```diff
@@ -0,0 +1,114 @@
+* With the high speed you can reach Barcelona in 2.5 hours.
+
+* From Sevilla Santa Justa you travel directly to Barcelona within 5.5 hours.
+
+### From France or Portugal
+
+From most bigger cities of France you can reach Barcelona within 7 to 9 hours.
+
+* From Paris, there is a direct TGV connection 3 times a day.
+* From Rennes, take the TGV via Nimes.
+* From Bordeaux, with InterCity and TGV via Narbonne.
+
+* From Lisboa Estacion Oriente there is a train to Madrid Chamartin. Since it is not a high speed train it takes 11 hours and you will probably need to continue on the second day.
+
+Note that there is a [cheaper Interrail country pass](https://www.interrail.eu/en/interrail-passes/one-country-pass/spain-passes) and you probably only need 3 days within 1 month.
+
+
+### From the North
+
+These events are timed so that people traveling towards Barcelona can stop by.
+What does that mean, let's go through a possible option:
+
+{:.layout-auto}
+| City                            | Day       | Departure | Arrival | Notes                                                                                     |
+|---------------------------------|-----------|-----------|---------|-------------------------------------------------------------------------------------------|
+| Gothenburg, SE                  | Wednesday | 08:55     | 12:28   | Swing by the office before and grab a laptop and a coffee to go.                          |
+| Kopenhagen, DK                  | Wednesday | 12:52     | 18:02   | Via Fredericia st.                                                                        |
+| Hamburg, DE                     | Wednesday | 18:02     |         | Here we could have a meetup in Hamburg, probably best with food.                          |
+| Hamburg, DE                     | Wednesday | 21:02     | 07:54   | Now we can fade out the day and rest.                                                     |
+| Basel, CH                       | Thursday  | 08:07     | 09:00   | This is the high speed train, you can stay on the NightJet and arrive a little later.      |
+| Zürich HB, CH                   | Thursday  |           | 09:42   | Time to walk a little in Zürich, meet people and maybe hit a shower and a day of hacking. |
+| Zürich HB, CH                   | Thursday  | 12:03     | 14:45   | At this point our group might have grown a bit.                                           |
+| Genève, CH                      | Thursday  | 14:59     | 18:05   | The fast TGV. 🚅                                                                          |
+| Valence TGV Rhône-Alpes Sud, FR | Thursday  | 18:22     | 22:44   | Say goodbye to the Alps.                                                                  |
+| Barcelona Sants, ES             | Thursday  |           | 22:44   | 🎉 Hurray, we made it.                                                                       |
+
+
+### From the East
+
+Now what if we were able to meet up with people?
```

Comment:
This post already has lots of rhetoric questions. I would just state this as a thing:

```suggestion
If your traveling from the east it might be possible to cross routes with folks from the north and join their travel.
```

or similar